### ISSUE TITLE
Weights manager: no selection of files by file extension for weights loading

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#203]
```
When selecting the Select Weights button, followed by the load button in
the Weights Manager dialog, the resulting list of files has all files, no 
highlight/selection
of gal and gwt weights only. In previous versions, the weights file extension 
was
detected to facilitate weights selection.
```

Original issue reported on code.google.com by `lanse...@gmail.com` on 22 Jul 2015 at 12:29

**Comment by lixun910** [comment]
fix is proposed in ci "#3728".

**Comment by jkoschinsky** [comment]
Verified in GeoDa 1.7.63: Only gwt and gal files lister in weights manager now.
